src/components/clusters/common/ScaleSection/MachineTypeSelection/MachineTypeSelection.tsx

```diff
@@ -204,14 +204,21 @@ const MachineTypeSelection = ({
   );
 
   const setDefaultValue = React.useCallback(() => {
-    const defaultType = cloudProviderID
+    const defaultTypeId = cloudProviderID
```

Comment:
Correct term (This variable uses the just id not the entire Type object)

Reply:
I wouldn't say `isTypeAvailable` has to be changed, it's id means the type and vise versa. This function checks if the Machine Type is available. It verifies it using it's id, given as an argument to the function
If really think it is important to be changed and consider it blocking, I can change it
Is it ?

Regarding other terminology changes, I believe I corrected all the ones that needed a change in this section
If you find any more, let me know 😄 Just don't waste too much time on it, I believe I already have 😅